docs/tutorials/developer-journey/level-5/5.5-SNS-tutorial.md

```diff
@@ -0,0 +1,1097 @@
+# 5.5 Launching an SNS
+
+## Overview
+
+A Service Nervous System (SNS) is an advanced form of a DAO. If you recall, DAOs and the NNS were covered in a previous module, [4.4 NNS and governance](../level-4/4.4-nns-governance.md). To recap, a DAO, or decentralized autonomous organization, is a form of decentralized governance that is controlled by the DAO's stakeholders without a central form of authority. The Internet Computer is managed by a DAO known as the Network Nervous System. 
+
+The underlying architecture of an SNS closely resembles the architecture of the NNS, but instead of being used to control the IC, an SNS is used to control a specific decentralized application that is running on the IC mainnet. Any dapp running on the mainnet can be handed over to an SNS, which results in the dapp being owned and controlled by a community. When an SNS takes control of a dapp, the dapp is then managed in the same way that the Internet Computer is managed by the NNS. When an SNS is launched, the ledger defines a unique token for the SNS that is then used by the community for participation in the SNS's governance. 
+
+The dapp's community can participate in the dapp's SNS through staking the SNS's native token in a neuron. Then, changes to the dapp are proposed to the community via an SNS proposal. SNS voting rights use the same format that voting on the NNS does; anyone can stake the SNS's native token into a neuron, and the neuron's voting power is proportional to the quantity of SNS tokens staked within the neuron, the length of their dissolve delay, and the amount of time since the neuron was created.
+
+## How an SNS works
+
+While the core architecture of an SNS resembles that of the NNS, the NNS contains additional canisters that provide important functionalities on the IC, such as the cycles minting canister that is responsible for creating cycles and the registry canister that contains the network's topology. In contrast, an SNS contains a few canisters that do not exist on the NNS, with the most notable canister being the decentralized swap canister used during the launch of the SNS. 
+
+SNSs are provided as a system functionality. For an SNS to be launched, the NNS community must vote on and approve the SNS's original canister code. Once the SNS is launched, the code for the SNS's canisters is maintained by the IC and is changed through SNS proposals that are voted on by the community. All approved SNS canister versions are stored on an NNS canister known as the SNS Wasm Modules canister (SNS-W). Recall that all canisters are compiled into Wasm before being deployed on the IC.
```

Comment:
This paragraph is a bit confusing to me since it seems to mix the SNS system canister and the dapp canisters that are controlled by a specific instance of a SNS.

Reply:
I agree that this might be a bit confusing.. How about:
> SNSs are provided as a system functionality. 
All SNS canister code is approved by the NNS community and thus all SNSs follow the same upgrade path. 
Launching a new SNS is also approved by the NNS community, including the initial settings of the SNS and the conditions of the decentralization swap.

(Maybe the part about the Wasms and where they are stored is going into too much detail and can also be omited?)

Reply:
Resolving this since the SNS tutorial has been removed per my conversation with @LaraAS.